TurkishFilter/sections/general_extensions.txt

```diff
@@ -27,6 +27,9 @@ fullhdfilmizlesene.org,ultrahdfilmizle.org#%#AG_onLoad(function() { window.start
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39485
 hintfilmcenneti.pw#%#(function(){var c=window.setInterval;window.setInterval=function(f,g){return g===1e3&&/_0x/.test(f.toString())&&(g*=0.01),c.apply(this,arguments)}.bind(window)})();
+! https://github.com/AdguardTeam/AdguardFilters/issues/39525
+roseline.club#%#//scriptlet("prevent-window-open")
+roseline.club#%#window.open = function() { };
```

Comment:
Both rules can be removed, this one is enough when I try. Please also add the domain to "puShown" rule ( AG_setConstant('puShown', 'true'); ).
||roseline.club/dmn.php